packages/axelar-local-dev/src/contracts/test/ExpressWithToken.sol

```diff
@@ -2,16 +2,16 @@
 pragma solidity 0.8.9;
```

Comment:
unrelated, but can we make all contracts in this repo to also have a floating ^0.8.0 pragma where possible for easier future maintenance?